tendrl/node_agent/tests/test_utils.py

```diff
@@ -0,0 +1,33 @@
+from mock import MagicMock
+import sys
+import tempfile
+sys.modules['logging'] = MagicMock()
```

Comment:
This is a good way to mock, but not the ideal please read more about "patch"

http://www.voidspace.org.uk/python/mock/patch.html#patch